libraries/index.html

```diff
@@ -0,0 +1,118 @@
+---
+layout: default
+title: Libraries
+permalink: /libraries
+---
+
+<div id="libraries-page" class="common-layout">
+  <div class="content">
+    <h1>CircuitPython Libraries</h1>
+    <p>
+      The CircuitPython Library Bundle contains all current libraries available for
+      CircuitPython. They are designed for use with CircuitPython and may or may not
+      work with MicroPython. The bundle options are explained below.
+    <p>
+      CircuitPython libraries are separate files designed to work with CircuitPython code.
+      CircuitPython programs require a lot of information to run. CircuitPython is so simple
+      to use because most of this information is processed in the background and stored in
+      libraries. Some libraries are built into CircuitPython. Others are downloaded and
+      stored on your CIRCUITPY drive in a folder called lib.
+    </p>
+    <p>
+      The full list of all the CircuitPython libraries contained in the following bundles
+      can be found on the
+      <a href="https://circuitpython.readthedocs.io/projects/bundle/en/latest/drivers.html">libraries page</a>.
+    <h2>Download and Installation Instructions</h2>
+    <p>
+      The libraries in each release are compiled to .mpy for all recent major versions
+      of CircuitPython. Please download the bundle that matches the major version
+      of your CircuitPython. For example, if you are running 4.0.0 you should
+      download the 4.x bundle. The .mpy files are designed to take up less space on
+      your CIRCUITPY drive. Unless you have a specific reason, choose the appropiate
+      .mpy library bundle!
+    </p>
+    <p>
+      To install, download the appropriate bundle for your version of CircuitPython.
```

Comment:
I'd like to make "To install" bold. It sort of disappears into the other paragraphs as it is.

Reply:
Should be good to go there now.